recipes/vcf2circos/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "vcf2circos" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "3cfd69f722f6f46246f972b917cf46c3af14aa0ccfcd7e029a63f0fb28defd79" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - "sed -i.bak '1 s|^.*$|#!/usr/bin/env python3|g' $PREFIX/bin/vcf2circos && pip install . --no-deps"
```

Comment:
Time order problem. You need to do the otherway around:

- First run pip to install vcf2circos
- Then patch it.

Because the bin/vcf2circos file doesn't exist yet at that point of time. This wrapper hasn't been created by pip yet.

```yaml
  script:
    - pip install . --no-deps"
    - sed -i.bak '1 s|^.*$|#!/usr/bin/env python3|g' $PREFIX/bin/vcf2circos
```

Reply:
Ah that makes sense! Thank you!